### ISSUE TITLE
Qualitative indicators introduced to the Project Editor 

### ISSUE THREAD

**Issue by AkvoAnt** [akvo/akvo-rsr#2756]
Frame 1: 

https://projects.invisionapp.com/share/6WCDMQTFR#/screens/241399804

**Comment by zzgvh** [comment]
With the introduction of qualitative indicators I would recommend that we enforce numerical values only for quantitative updates.

To see if this would be hard, I checked how many updates we currently have that would need "fixing", i.e. they are updates with non-numerical values. The results look like this


Value | Count
-----|-----
Empty string  | 7
Hyphen (-) | 5
Available | 9
Gender aware | 2
Gender neutral | 2
N/A | 49
See description | 4
Yes | 1
available | 1
yes | 1

using this SQL:
```sql
SELECT
  u.data,
  count(u.data)
FROM rsr_project p, rsr_result res, rsr_indicator ind, rsr_indicatorperiod per,
  rsr_indicatorperioddata u
WHERE u.data !~ '[0-9]' AND res.project_id = p.id AND ind.result_id = res.id AND
      per.indicator_id = ind.id AND u.period_id = per.id
GROUP BY u.data
ORDER BY u.data
```
(The SQL can be easily modded to show which projects, results etc the updates belong to)

The Value column is the text entered as data for the update, the Count column is how many updates have this exact value. Not that the first value is an empty string, and the second is only a hyphen, "-", but markdown mess them up 😉. The results above suggest that it should be easy to migrate the values since most of them are an indication that there is no value. The few that hold any data can be moved to the text field for the update.

**Comment by zzgvh** [comment]
@punchagan I've added an initial version of different update forms for quantitative and qualitative updates. I'm not sure if this is the "right" way, there is a fair amount of code duplication in the components that render the form. OTOH it's easy to compose the final form from parts this way. Not sure how well it'll work when we add percentage and disaggregation to the mix.

**Comment by zzgvh** [comment]
Running this SQL:

```sql
SELECT DISTINCT
  org.name, p.id, upd.data
FROM rsr_project p, rsr_result res, rsr_indicator ind, rsr_indicatorperiod per,
  rsr_indicatorperioddata upd, rsr_partnership par, rsr_organisation org
WHERE upd.data !~ '[0-9]' AND res.project_id = p.id AND ind.result_id = res.id AND
      per.indicator_id = ind.id AND upd.period_id = per.id and par.project_id = p.id and par.organisation_id = org.id
ORDER BY org.name, upd.data, p.id
```

returns the following (Note that there are more records here than above since more than one organisation may be connected to a project and thus to a particular data value.):

Org name | Project ID | data
-----|-----|-----
ADOPEM | 3077 | N/A
AFORD Foundation | 3988 | See description
AFSRT | 3983 | See description
AMREF Kenya | 3981 | yes
AMREF Uganda | 3983 | See description
AST | 2977 | N/A
Akvo | 2780 | 
Akvo | 2212 | Available
Akvo | 2213 | Available
Akvo | 2780 | Available
Akvo | 2783 | Available
Akvo | 2849 | Available
Akvo | 3992 | N/A
Akvo | 3979 | See description
Akvo | 3983 | See description
Akvo | 3988 | See description
Akvo | 3992 | See description
Akvo | 3979 | Yes
Akvo | 2783 | available
Akvo | 3981 | yes
Akvo India | 4203 | 
Amref Flying Doctors | 3992 | N/A
Amref Flying Doctors | 3983 | See description
Amref Flying Doctors | 3992 | See description
Amref Flying Doctors | 3981 | yes
Anonymous | 3816 | N/A
Bedari | 2778 | -
Belle Porte | 2977 | N/A
Both ENDS | 3979 | See description
Both ENDS | 3979 | Yes
CBM | 2977 | N/A
CLIP | 3988 | See description
CONANI | 2937 | N/A
Coalition for Children | 2937 | N/A
DORP | 3979 | See description
DORP | 3979 | Yes
FODDET | 2977 | N/A
FUNDAPEC | 3077 | N/A
Fetaph | 2977 | N/A
Finland MFA | 2938 | Gender neutral
Finland MFA | 2937 | N/A
Finland MFA | 2938 | N/A
Finland MFA | 2940 | N/A
Finland MFA | 3077 | N/A
Finland MFA | 3299 | N/A
GYAM | 3988 | See description
Ghana WASH Alliance  | 3988 | See description
Government of Finland | 2778 | -
Government of Finland | 2952 | Gender aware
Government of Finland | 3086 | Gender aware
Government of Finland | 3768 | Gender neutral
Government of Finland | 2952 | N/A
Government of Finland | 2977 | N/A
Government of Finland | 3086 | N/A
Government of Finland | 3757 | N/A
Government of Finland | 3816 | N/A
Government of Finland | 3896 | N/A
HEWASA | 3983 | See description
HUNDEE | 2940 | N/A
Handicap International | 2977 | N/A
Hilton Foundation | 2952 | Gender aware
Hilton Foundation | 2952 | N/A
INADES | 3299 | N/A
INFOTEP | 3077 | N/A
INTAGRAD | 3988 | See description
IRC | 3992 | N/A
IRC | 3992 | See description
IRM | 2778 | -
JESE | 3983 | See description
KEWASNET | 3981 | yes
KMG-Ethiopia | 2940 | N/A
MOPAFEJ | 3299 | N/A
Mbarara University | 3757 | N/A
McGill University | 3757 | N/A
Monde Radieux | 2977 | N/A
NAWAD | 3983 | See description
NIA | 3981 | yes
NL Min. Foreign Affairs | 3992 | N/A
NL Min. Foreign Affairs | 3992 | See description
NLC - consulting | 3299 | N/A
NewEnergy | 3988 | See description
OKANI | 3299 | N/A
PADet | 2940 | N/A
Plan Dominican Republic | 2937 | N/A
Plan Dominican Republic | 3077 | N/A
Plan Ethiopia | 2938 | Gender neutral
Plan Ethiopia | 2938 | N/A
Plan Ethiopia | 2940 | N/A
Plan Finland | 2778 | -
Plan Finland | 2952 | Gender aware
Plan Finland | 3086 | Gender aware
Plan Finland | 2938 | Gender neutral
Plan Finland | 3768 | Gender neutral
Plan Finland | 2937 | N/A
Plan Finland | 2938 | N/A
Plan Finland | 2940 | N/A
Plan Finland | 2952 | N/A
Plan Finland | 2977 | N/A
Plan Finland | 3077 | N/A
Plan Finland | 3086 | N/A
Plan Finland | 3299 | N/A
Plan Finland | 3757 | N/A
Plan Finland | 3816 | N/A
Plan Finland | 3896 | N/A
Plan Kenya | 2952 | Gender aware
Plan Kenya | 2952 | N/A
Plan Mozambique | 3896 | N/A
Plan Pakistan | 2778 | -
Plan Pakistan | 3768 | Gender neutral
Plan Sweden | 2940 | N/A
Plan Timor-Leste | 3086 | Gender aware
Plan Timor-Leste | 3086 | N/A
Plan Togo | 2977 | N/A
Plan Uganda | 3757 | N/A
Plan Uganda | 3816 | N/A
Practical Action Kenya    | 3981 | yes
Presby | 3988 | See description
RAIN | 3992 | N/A
RAIN | 3983 | See description
RAIN | 3992 | See description
RAIN | 3981 | yes
RATSON | 2938 | Gender neutral
RATSON | 2938 | N/A
RUAF | 3988 | See description
SAATHI | 4203 | 
SASOL Foundation | 3981 | yes
SEDUC | 3299 | N/A
SIDA | 2940 | N/A
SIL | 3299 | N/A
SLOPB | 3979 | See description
SLOPB | 3979 | Yes
STVM | 3086 | Gender aware
STVM | 3086 | N/A
Simavi | 4323 | 
Simavi | 3992 | N/A
Simavi | 3979 | See description
Simavi | 3983 | See description
Simavi | 3988 | See description
Simavi | 3992 | See description
Simavi | 3979 | Yes
Simavi | 3981 | yes
Simli AiD | 3988 | See description
Titan CSR | 4203 | 
UNDP (duplicate) | 2977 | N/A
UNICEF | 2977 | N/A
URWA | 3983 | See description
USSIA | 3983 | See description
Urbanet | 3988 | See description
Uttaran | 3979 | See description
Uttaran | 3979 | Yes
WASH Alliance Bangladesh | 4323 | 
WASH Alliance Bangladesh | 3979 | See description
WASH Alliance Bangladesh | 3979 | Yes
WASH Alliance International | 4323 | 
WASH Alliance International | 3981 | yes
WASH Alliance Kenya | 3981 | yes
WASH alliance uganda | 3983 | See description
WASTE | 3992 | N/A
WASTE | 3979 | See description
WASTE | 3983 | See description
WASTE | 3988 | See description
WASTE | 3992 | See description
WASTE | 3979 | Yes
WASTE | 3981 | yes
WODA | 2938 | Gender neutral
WODA | 2938 | N/A
WaterAid Bangladesh | 4323 | 
WaterAid Bangladesh | 3979 | See description
WaterAid Bangladesh | 3979 | Yes
Wetlands International | 3992 | N/A
Wetlands International | 3983 | See description
Wetlands International | 3992 | See description
Ylen Hyvä | 3086 | Gender aware
Ylen Hyvä | 3086 | N/A

**Comment by punchagan** [comment]
Closed with #2791
